sheet2linkml/cli.py

```diff
@@ -127,6 +160,8 @@ def main(output, filter_entity, logging_config, write_mappings, include_terminol
         if write_mappings:
             Mappings.write_to_file(model.mappings, filename=write_mappings, model=model)
 
+    exit(0)
```

Comment:
Feel free to modify this to `sys.exit()` based on the recommendation above.

Reply:
Fixed in 8ebf86e.